src/controllers/notifications.controller.js

```diff
@@ -31,5 +31,30 @@ class NotificationController {
         return res.status(404).json({ error: 'route does not exist' });
     }
   }
+
+  /**
+ *
+ * @description Method to opt in and out of email and push notication
+ * @static
+ * @param {object} req client request
+ * @param {object} res server response
+ * @returns {Object} server response object
+ * @param  {Function} next passes control to the next middleware
+ * @memberof NotificationController
+ */
+  static async getUserNotification(req, res) {
+    try {
+      const user = req.auth;
+      const userNotification = await models.AppNotification.findAll({
```

Comment:
please consider if notification is new or the user already seen it.

Reply:
Fixed

Reply:
how I can't see it